tempesta_fw/pool.c

```diff
@@ -70,21 +89,22 @@ EXPORT_SYMBOL(tfw_pool_alloc);
 void *
 tfw_pool_realloc(TfwPool *p, void *ptr, size_t old_n, size_t new_n)
 {
-	unsigned char *p_tmp = ptr;
+	unsigned char *tmp_ptr = ptr;
 	void *a;
+	TfwPoolChunk *chunk;
+
```

Comment:
It's good to call here ```try_free()``` function (https://github.com/natsys/tempesta/issues/188#issuecomment-138092421): if ```ptr``` is the last allocated item, then we can reuse it's memory. So it's good to implement ```try_free``` at the same time.